lib/commands/signout.js

```diff
@@ -0,0 +1,23 @@
+const { SignOut } = require('../messages/flow');
+
+const SignedParams = require('../signed-params');
+
+module.exports = async (req, res) => {
+  await res.locals.slackUser.update({
+    githubId: null,
+  });
+
+  const { command } = res.locals;
+
+  const state = new SignedParams({
+    trigger_id: command.trigger_id,
+  });
+
+  const protocol = req.headers['x-forwarded-proto'] || req.protocol;
+  const host = req.headers['x-forwarded-host'] || req.get('host');
```

Comment:
What do you think about pulling this out into a module? I think we do this in 2-3 places now.